application-mocca-calendar-ui/src/main/resources/MoccaCalendar/Code/MoccaCalendarEventModificationClass.xml

```diff
@@ -208,34 +208,56 @@
       <cache>long</cache>
     </property>
     <property>
-      <code>(function() {
+      <code>require(['moment'], function(moment) {
 
   function disable(element) { element.setAttribute('disabled','disabled') }
   function enable(element) { element.removeAttribute('disabled') }
 
+  function setOriginalDateFormat() {
+    var endDateInput = document.querySelector('input[id^="MoccaCalendar\\.Code\\.MoccaCalendarEventModificationClass_"][id$="_endDate"]');
```

Comment:
I see you are mixing `const`, `let` and `var` declarations in this method. You could check https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/ if you're not yet very sure on their usage